src/mSupplyMobileApp.js

```diff
@@ -283,10 +223,19 @@ class MSupplyMobileAppContainer extends React.Component {
     );
   };
 
+  getOptions = () => ({
```

Comment:
these options are 'defaults' for screens within the navigator. Only called once. Looking for a better place to put it.